src/app/page/page.component.ts

```diff
@@ -115,6 +117,7 @@ export class PageComponent implements OnInit, OnDestroy {
     this.route.queryParams.subscribe((queryParam) => {
       this.embedded = queryParam.embedded === 'true';
     });
+    this.activePageType = this.setPageType(this.activePage);
```

Comment:
Instead of making an instance variable, I was thinking we could call `this.getPageType()` from within the template. Is that doable?